mkdocs.yml

```diff
@@ -46,6 +46,7 @@ nav:
           - AudioSegmentResource: xmllib-api-reference/dsp-base-resources/audio-segment-resource.md
       - Configuration options: xmllib-api-reference/config-options.md
       - Value Checkers: xmllib-api-reference/value-checkers.md
+      - Value Converters: xmllib-api-reference/value-converters.md
```

Comment:
I think it is helpful to have these here too, so they can see the source-code of the converters we apply if they want.

Reply:
Well, if it is really only about that, then they can look at the source code themselves. Adding this to the documentation conveys the message: "This is important for you, you can use these". If you don't want to convey this message, then I wouldn't add the converters to the docs.

Reply:
Let's leave it for now and if they (or noemi) does not like it we can always remove it easily.